src/components/learner-credit-management/cards/status-modals/ContentNotInCatalogErrorAlertModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { AlertModal, ActionRow, Button } from '@edx/paragon';
+import { Error } from '@edx/paragon/icons';
+
+import { commonErrorAlertModalPropTypes, getBudgetDisplayName } from '../data';
+import { useBudgetId, useSubsidyAccessPolicy } from '../../data';
+
+const ContentNotInCatalogErrorAlertModal = ({
+  isErrorModalOpen,
+  closeErrorModal,
+  closeAssignmentModal,
+}) => {
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+
+  const budgetDisplayName = getBudgetDisplayName(subsidyAccessPolicy);
+
+  const handleClose = () => {
+    closeErrorModal();
+    closeAssignmentModal();
+  };
+
+  return (
+    <AlertModal
+      isBlocking
+      variant="danger"
+      icon={Error}
+      title={`This course is not in your ${budgetDisplayName}'s catalog`}
```

Comment:
non-blocking grammar question/proposal: including the word "your" here assumes that a display name will typically end with the word "budget".  Maybe the default display name should be "your budget" and we switch the usage a little bit in these strings to be like "This course is not in the catalog for ${budgetDisplayName}".

Reply:
@iloveagent57 [clarification] The `getBudgetDisplayName` function will always does end with the work "budget":

```jsx
// src/components/learner-credit-management/cards/data/utils.js

export const getBudgetDisplayName = (subsidyAccessPolicy) => {
  let budgetDisplayName = 'budget';
  if (subsidyAccessPolicy.displayName) {
    budgetDisplayName = `${subsidyAccessPolicy.displayName} ${budgetDisplayName}`;
  }
  return budgetDisplayName;
};
```

If a policy has a display name, the title will be "This course is not in your <display_name> budget's catalog". If a policy does not have a display name, the title would read "This course is not in your budget's catalog."

I guess I'm not entirely following your suggestion? 🤔